src/isomorphic/devtools/ReactComponentTreeDevtool.js

```diff
@@ -19,6 +19,7 @@ var rootIDs = [];
 function updateTree(id, update) {
   if (!tree[id]) {
     tree[id] = {
+      source: null,
```

Comment:
While we’re at it, we might as well store the `element` itself because React DevTools will want to know it.